nixos/modules/services/admin/nomachine.nix

```
@@ -0,0 +1,258 @@
+{ config, lib, pkgs, ... }:
+with lib;
+let
+  cfg = config.services.nxserver;
+  nxserver = pkgs.nomachine;
+
+  preStartScript = pkgs.writeScript "nxserver-prestart" ''
+    #!${pkgs.runtimeShell}
+
```

```suggestion
    set -euo pipefail

```
To make the unit fail and stop if something goes wrong.